src/lib/external/aura/getAuraPools.ts

```diff
@@ -0,0 +1,57 @@
+interface AuraPool {
+  id: string;
+  isShutdown: boolean;
+  aprs: {
+    total: number;
+    breakdown: {
+      id: string;
+      token: {
+        symbol: string;
+        name: string;
+      };
+      name: string;
+      value: number;
+    }[];
+  };
+  lpToken: {
+    address: string;
+  };
+}
+
+export default async function getAuraPools(chainId: number): Promise<AuraPool[]> {
+  const res = await (await fetch('https://data.aura.finance/graphql', {
```

Comment:
I've been using `axios` for HTTP requests. We should stick to one of them. Don't have a real preference.